Safety/contact-numbers.rst

```diff
@@ -0,0 +1,124 @@
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+###############
+Contact Numbers
+###############
+
+These extensions can be dialed from any telephone in the Rubin Control room or on the Rubin summit. 
+They are internal to AURA.
+
+.. _contact-numbers-emergency-numbers:
+
+Emergency Numbers
+^^^^^^^^^^^^^^^^^
+.. Preconditions
+
+- Polyclinic Paramedic: 577
+
+- Mountain Assistant/Asistante Montaña: 440
+
+- Main gate guards/Control Puerta: 434
+
+.. note::
+    Staff working these positions primarily speak Spanish.
+    If no one answers the telephone at any of the extensions above, use any Rubin radio tuned to Channel 1. 
+    Put out this call: 
+    
+    - "Atento [paramedico/asistante montaña/control puerta] - habla [your name] del Observatorio Rubin" and await a response.
+
+
+.. _contact-numbers-non-emergency-numbers:
+
+Non-Emergency Numbers
+^^^^^^^^^^^^^^^^^^^^^
+
+- Rubin Casino: 593
+
+  Staff primarily speak Spanish.
+
+- SOAR Control Room: 500
+  
+  Staff primarily speak Spanish, but some operators are bilingual.
+
+- Gemini Base Facility (operations control room) in La Serena: 758
+  
+  Staff are bilingual.
+
+.. _contact-numbers-calling-hotel-pachon:
+
+
+Calling Hotel Pachon
+^^^^^^^^^^^^^^^^^^^^
+
+If there is something wrong with the room you have organized to stay in, you may call 440 from the hotel foyer and request help.
+Some examples are if the key is not hanging on the second floor for you, there is no hot water, or the heat is off and you want to request a space heater, etc.
+Staff mostly speak Spanish only.
+
+- 20-Units main telephone: 599
+
+- Hotel Pachon Indiviual Room Numbers
+
+  - Room 1: 571
+  - Room 2: 572
+  - Room 3: 573
+  - Room 4: 574
+  - Room 5: 575
+  - Room 6: 576
+  - Room 7: 589
+  - Room 8: 590
+  - Room 9: 587
+  - Room 10: 578
+  - Room 11: 579
+  - Room 12: 580
+  - Room 13: 581
+  - Room 14: 582
+  - Room 15: 555
+  - Room 16: 556
+  - Room 17: 557
+  - Room 18: 558
+
+.. _contact_numbers_using_the_control_room_phones:
+
+Using the control room phones 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. note::
+  When dialing a large number, leave the phone in the cradle and dial the number using the keypad. 
+  Then pick up the phone after you have entered the full number.
+
+- To dial a landline in Chile: 9 2 XXXX XXXX
+
+- To dial a cell phone in Chile: 9 9 XXXX XXXX
+
+- To dial the United States: 001 + Area Code + Phone Number
+
+- To dial another country: 9 + 188 + 0 + Country Code + Area Code + Phone Number
+
+.. _Daytime-Operations-Safety-contact-numbers-contact-personnel:
+
+Contact Personnel
+^^^^^^^^^^^^^^^^^
+
```

Comment:
This is empty on purpose?
If so, maybe comment out the title?
Or, if you want it to be there for a place holder, add some generic sentence (e.g. Section content to be completed in a future update)

Reply:
The text is there ...